modules/web/src/app/wizard/step/cluster/ssh-keys/component.ts

```diff
@@ -134,6 +134,11 @@ export class ClusterSSHKeysComponent extends BaseFormValidator implements OnInit
     return !_.isEmpty(this.keys);
   }
 
+  writeValue(keys: SSHKey[]): void {
+    this.form.get(Controls.Keys).setValue(keys, {emitEvent: false});
+    this.keys = keys;
```

Comment:
I think `this.keys` are SSH keys of selected project (and not selected keys) so we shouldn't override them here or am I missing something?